zproject/settings.py

```diff
@@ -1302,6 +1302,6 @@ def get_secret(key):
 # This is a debugging option only
 PROFILE_ALL_REQUESTS = False
 
-CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
+CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com', 'welcome-bot@zulip.com'))
```

Comment:
@rishig I think welcome bot can't send PM users which are not in realm Zulip. I tried sending the message  using `internal_prep_private_message` but it didn't went through. Welcome bot can send messages to streams in other realms as there is a special exception for welcome-bot in check_message function in actions.py when it comes to sending messages to streams.

```
 elif sender.is_bot and (sender.bot_owner is not None and
                                subscribed_to_stream(sender.bot_owner, stream)):
            # Or you're a bot and your owner is subscribed.
            pass
        elif sender.email == settings.WELCOME_BOT:
            # The welcome bot welcomes folks to the stream.
            pass
``` 

I can add an exception for welcome-bot in the case of private messaging so that It can send PMS to users in others realms.